### PR TITLE
Reproduce rounding issue with 7% validator fee

### PR DESCRIPTION
**Pull request by lukaw3d**
Reproduces https://github.com/oasisprotocol/oasis-wallet-web/issues/436

### CODE REVIEW

File: src/app/pages/StakingPage/Features/DelegationList/__tests__/__snapshots__/ActiveDelegationList.test.tsx.snap [hunk 1]
```diff
@@ -604,7 +604,7 @@ Object {
                     id="cell-fee-undefined"
                     role="gridcell"
                   >
-                    10%
+                    7.000000000000001%
```
Comment (lukaw3d) [inline src/app/pages/StakingPage/Features/DelegationList/__tests__/__snapshots__/ActiveDelegationList.test.tsx.snap:607]: <

**Comment by matevz** [comment]
Interesting bug :thinking: While you're at it, can you also round the values in the Fee column in validator list to 2 decimals or so (validators could  use some crazy fee just to spoil the UI) and maybe align the text to right :wink:

**Comment by pro-wh** [comment]
let's round to the commission denominator. that's thousandths of a percent https://github.com/oasisprotocol/oasis-core/blob/v21.3.6/go/staking/api/commission.go#L19

**Review by pro-wh — APPROVED** [review]
let's merge this so that we'll have a diff in the snapshot if something fixes this

**Comment by lukaw3d** [comment]
Okay :shrug:

**Pull request merged**